### PR TITLE
Makes the Ai braindead message not appear if the AI shunts

### PR DESCRIPTION
**Pull request by Shadowlight213**

### CODE REVIEW

**Comment by Aranclanos** [comment]
ehh its a really poor fix

File: code/modules/mob/living/silicon/ai/examine.dm [hunk 1]
```diff
@@ -18,7 +18,7 @@
 		if (src.stat == UNCONSCIOUS)
 			msg += "It is non-responsive and displaying the text: \"RUNTIME: Sensory Overload, stack 26/3\".\n"
 		msg += "</span>"
-		if (!src.client)
+		if (src.shunted == 0 && !src.client)
```
Comment (RemieRichards) [inline code/modules/mob/living/silicon/ai/examine.dm]: pls no ```src```, it's unneeded as it's implied!

**Comment by Shadowlight213** [comment]
@aranclanos I'm using the var in case someone else wants to add another feature based off of it, as the shunting code is done APC side. I asked jordie about it and he says it was a good idea.

File: code/modules/mob/living/silicon/ai/examine.dm (hunk 1 above)
Comment (Shadowlight213) [inline code/modules/mob/living/silicon/ai/examine.dm]: Oh. I didn't know that.

**Comment by Jordie0608** [comment]
You really should just check if a client is in the AI core or not.

**Comment by Anonus** [comment]
Uh, Jordie it already does that?

**Comment by Jordie0608** [comment]
I may have entirely misread everything here.

**Comment by Shadowlight213** [comment]
@Jordie0608 It originally just checked for client, but since shunting transfers the mind to the APC, it would show braindead.

**Comment by phil235** [comment]
could be a bit better, but still satisfactory.

**Comment by Aranclanos** [comment]
horrible

**Pull request merged**